src/drivers/net/intel.h

```diff
@@ -200,6 +200,14 @@ struct intel_descriptor {
 #define INTEL_FEXTNVM11 0x05bbcUL
 #define INTEL_FEXTNVM11_WTF	0x00002000UL	/**< Don't ask */
 
+/** I210 Packet Buffer Sizes */
+#define INTEL_E1000_RXPBS	0x02404
+#define INTEL_E1000_TXPBS	0x03404
+#define INTEL_I210_DTXMXPKTSZ	0x355C
+#define INTEL_I210_RXPBSIZE_DEFAULT	0x000000A2
+#define INTEL_I210_TXPBSIZE_DEFAULT	0x04000014
+#define INTEL_I210_DTXMXPKTSZ_DEFAULT	0x00000098
```

Comment:
Happy for these to retain `I210` within the name, since the values are I210-specific.